.github/workflows/backport-stale.yml

```diff
@@ -0,0 +1,29 @@
+name: Mark stale backport requests
+
+on:
+  schedule:
+  - cron: "0 10 * * 1-4"
+  workflow_dispatch:
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+    - uses: actions/stale@v3
+      with:
+        operations-per-run: 1000
```

Comment:
so i guess there's no easy way to make this rule send a reminder regularly? what if we don't add a `no-backport-pr-activity`? will it just keep reminding every 3 weeks?

Reply:
It will not, it will use a default label. If you remove the label it will remind again.